src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -458,19 +458,17 @@ def _augment_measured_parameters(self, measured_parameters):
             if isinstance(value, list):
                 value = json.dumps(value)
 
-            data_type = python_type_to_xml_type(value)
-
-            # TODO: This is hardcoded for now, but we should eventually try to guess this or add it to the descriptions
-            #  YAML file
-            attr_type = 'processingInformation'
+            guessed_data_type = python_type_to_xml_type(value)
+            guessed_attr_name = (name.title()
```

Comment:
Couple of things I would like to see here: lets lose the `.replace('Mgrs', 'MGRS').replace('Dswx', 'DSWx')` in favor of the new display name override field in the config for any of the parameters that include 'DSWx' or 'MGRS'.

For the remaining transformation (name.title().replace('_', '')`, lets factor this out into a simple utility function that we can reuse when we roll out this feature to the other PGEs.

Reply:
Where would I put this util function?

Somewhere under `src/opera/util/` perhaps but in which module?

Reply:
Since its related to rendering of the metadata, `util/render_jinja2.py` is as good a spot as any